examples/window/screenshot.rs

```diff
@@ -13,17 +12,16 @@ fn main() {
 }
 
 fn screenshot_on_spacebar(
+    mut commands: Commands,
     input: Res<ButtonInput<KeyCode>>,
-    main_window: Query<Entity, With<PrimaryWindow>>,
-    mut screenshot_manager: ResMut<ScreenshotManager>,
     mut counter: Local<u32>,
 ) {
     if input.just_pressed(KeyCode::Space) {
         let path = format!("./screenshot-{}.png", *counter);
         *counter += 1;
-        screenshot_manager
-            .save_screenshot_to_disk(main_window.single(), path)
-            .unwrap();
+        commands
+            .spawn(Screenshot::primary_window())
+            .observe(save_to_disk(path));
```

Comment:
Would be really cool to extend this example to print out "screenshot saved to {}" on success to also demonstrate how to get that info.

Reply:
We do print that out inside the `save_to_disk` callback... I think it's a helpful utility but anyone doing it "for real" probably just want's to roll their own. However, I think it's a good idea to demonstrate a different type of observer, so will do that 👍